src/matrix/calls/group/GroupCall.ts

```diff
@@ -125,6 +127,7 @@ export class GroupCall extends EventEmitter<{change: never}> {
     }
 
     get localMedia(): LocalMedia | undefined { return this.joinedData?.localMedia; }
+    get localPreviewMedia(): LocalMedia | undefined { return this.joinedData?.localPreviewMedia; }
```

Comment:
I think we can remove the `localMedia` getter. It should only be used from previewing in the UI.

Reply:
It's used in CallViewModel.toggleCamera() and toggleMicrophone() to check if tracks are available